src/jit/gentree.h

```diff
@@ -120,6 +120,8 @@ enum genTreeKinds
     GTK_NOVALUE = 0x0400, // node does not produce a value
     GTK_NOTLIR  = 0x0800, // node is not allowed in LIR
 
+    GTK_NOCONTAIN = 0x1000, // this node is a value, but may not be contained
```

Comment:
Could we just add a new GT opcode that means contained mem?  It looks like you could combine IND, LEA, and LCL_VAR into one "contained" mem thing at lower and be done with it.    The reason I'm thinking new opcode is that the register requirements would different in that the operation would not produce a value.  I'm also more inclined to have this at lower so that a more aggressive and concrete job could be done building address modes before RA.